boa/src/syntax/ast/op.rs

```diff
@@ -40,15 +40,16 @@ impl Display for NumOp {
                 NumOp::Sub => "-",
                 NumOp::Div => "/",
                 NumOp::Mul => "*",
-                NumOp::Pow => "**",
+                NumOp::Exp => "**",
                 NumOp::Mod => "%",
             }
         )
     }
 }
 
 #[derive(Clone, Debug, Trace, Finalize, PartialEq)]
-/// A unary operation on a single value
+/// A unary operation on a single value   
```

Comment:
```suggestion
/// A unary operation on a single value
```

Could this be due to a missing `cargo fmt`?

Reply:
oh, maybe, sometimes it runs on save and sometimes it doesn't (i think if there's errors)

Reply:
This is intentional.

Reply:
If there's a better way of doing this im open to it, ive just found adding spaces is the only way to get it to work for me

Reply:
You can use `\` (backslash) at the end with no trailing spaces, to do a newline. Even though it is not a part of the markdown  specification. `cargo doc` markdown parser parses it as a newline, which is what we want.

For Example:
```text
hello\
world
```
_**is:**_
hello\
world

Here is the comment I got the idea from:
https://gist.github.com/shaunlebron/746476e6e7a4d698b373#gistcomment-2271765

Reply:
Or alternatively you can just use `<br>`, since in markdown you can use HTML.

Reply:
`\` seems to work with rust-analyzer too, Thanks im happy to use this in future

Reply:
I added a suggestion to improve this. If we use the idiomatic way of Rust to display a summary in one line and the explanation in the second one, we can forget about this limitation :)

Reply:
> I added a suggestion to improve this

I might have missed this, but what was the suggestion?

Reply:
> > I added a suggestion to improve this
> 
> I might have missed this, but what was the suggestion?

I think the diff is outdated here, feel free to resolve the conversation. I think the new pattern I used when documenting other functions of the parser was to add the small summary in the first line, then an empty line, and the rest in the next lines.

This is similar to how the Rust standard library does it: https://doc.rust-lang.org/stable/src/std/path.rs.html#2446-2470